channeldb/channel.go

```diff
@@ -3579,6 +3606,18 @@ func serializeChannelCloseSummary(w io.Writer, cs *ChannelCloseSummary) error {
 		}
 	}
 
+	// Write whether the local force close info is present.
+	if err := WriteElements(w, cs.LocalFCInfo != nil); err != nil {
```

Comment:
This won't work without changes to the `WriteElement` and `ReadElement` functions in `channeldb/codec.go`.

Reply:
Why do you say this? `bool` is handled by `WriteElement`.

Reply:
That is the general pattern of writing it looking at the function.